hosts_example.yml

```diff
@@ -166,6 +166,89 @@ all:
     #   properties:
     #     key: val
 
+    #### RBAC with Mutual TLS authentication Configuration with 3 listeners ####
```

Comment:
Should just be RBAC, there are already TLS docs, make sure this section matches whats in the gdoc

Reply:
resolving in this pr: https://github.com/confluentinc/cp-ansible/pull/252